src/app/pages/loading-waiting-states/skeleton-loader/components/placeholder-score-card/placeholder-score-card.component.scss

```diff
@@ -80,7 +87,8 @@
         .footer-content {
             display: flex;
             padding: 16px;
-            color: map-get($pxb-gray, 500);
+            color: map-get($pxb-gray, 200);
+            height: 3.25rem;
         }
```

Comment:
Maybe use an info list item here and set it to `disabled` so that you wouldn't have these styles